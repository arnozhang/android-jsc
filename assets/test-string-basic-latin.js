function test () {
  return 'abc def {|} 0Oo.'
}
