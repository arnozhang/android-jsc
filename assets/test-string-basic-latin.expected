abc def {|} 0Oo.
